Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.yml

```diff
@@ -33,6 +33,7 @@ configuration:
   display: Minimum incident severity to send notifications to Teams by
   name: min_incident_severity
   options:
+  - Unclassified
```

Comment:
I think the term is `Unknown` but doesn't really matter

Reply:
fixed